packages/blockchain-wallet-v4-frontend/src/data/analytics/types/viewAndClick.ts

```diff
@@ -1,8 +1,26 @@
 // View and Click Events
 export enum Events {
+  BANNER_REWARDS_CLICKED = 'Banner Rewards Clicked',
+  BANNER_REWARDS_VIEWED = 'Banner Rewards Viewed',
   VIEW_AND_CLICK_FIAT_CURRENCY_SELECTED = 'Fiat Currency Selected'
 }
 
+type BannerEarnRewardsButtonView = {
+  key: Events.BANNER_REWARDS_VIEWED
+  properties: {
+    device: 'APP-Android' | 'APP-iOS' | 'WEB'
+    platform: 'EXCHANGE' | 'WALLET'
+  }
+}
+
+type BannerEarnRewardsButtonClick = {
+  key: Events.BANNER_REWARDS_CLICKED
+  properties: {
+    device: 'APP-Android' | 'APP-iOS' | 'WEB'
+    platform: 'EXCHANGE' | 'WALLET'
+  }
+}
```

Comment:
dont need to define these properties since that is handled for you by the wrapper and we will never change them

Reply:
Done, thanks andrew!